alibi/utils/tests/test_distributed.py

```diff
@@ -2,13 +2,17 @@
 import os
 import pytest
 import time
+from itertools import product
+from typing import Any, Dict, List, Optional, Tuple, Union
 
 import numpy as np
 
+# Skip distributed tests on Windows for now (Windows support for ray is experimental
+# https://docs.ray.io/en/latest/installation.html#windows-support), see CI for more info.
+ray = pytest.importorskip('ray')
```

Comment:
Not sure if this import is necessary given that each of the test functions also calls this? Or perhaps we can get rid of the calls inside the test functions (tho not sure `pytest` will skip those using this module-level import?).

Reply:
I've removed the calls inside the test functions and just kept the one above